components/authz-service/storage/postgres/migration/legacy/iam.go

```diff
@@ -254,7 +252,7 @@ func legacyPolicyFromV1(pol *v1Policy) (*v2Policy, error) {
 
 	if pol.ID.String() == constants_v1.IngestWildcardPolicyID {
 		ingestStatement := newV2Statement(Allow, "", []string{},
-			[]string{"*"}, []string{"infra:ingest:*"})
+			[]string{"*"}, []string{"ingest:*"})
```

Comment:
Since we have the opportunity to allow this file to represent "the state of the world" at force-upgrade, we've modified the v1 legacy migration. In the future, this file should be left untouched.